### ISSUE TITLE
Variable Explorer gives an error when trying to see a DataFrame

### ISSUE THREAD

**Issue by lemontreeshy** [spyder-ide/spyder#2866]
```python-traceback
>>> Traceback (most recent call last):
  File "//anaconda/envs/py27/lib/python2.7/site-packages/spyderlib/widgets/dataframeeditor.py", line 445, in copy
    col_min, col_max) = get_idx_rect(self.selectedIndexes())
  File "//anaconda/envs/py27/lib/python2.7/site-packages/spyderlib/widgets/arrayeditor.py", line 96, in get_idx_rect
    rows, cols = list(zip(*[(i.row(), i.column()) for i in index_list]))
ValueError: need more than 0 values to unpack
```

my code is 

```python
import pandas as pd

assets_temp = []
assets_temp.append([1,0,0.06,500000,0,"02","01","02",1])
assets_temp.append([2,100,0.09,500000,50000000,"01","01","02",1])
assets_temp.append([3,0,0.035,50000,500000000,"02","01","02",1])
assets_temp.append([4,10,0.07,500000,0,"01","01","01",1])
assets_temp.append([5,30,0.08,500000,0,"01","01","02",1])
#%%
asset_clomuns=['asset_id',
               'asset_period',
               'asset_profit',
               'asset_min_amt',
               'asset_max_amt',
               'asset_type',
               'asset_status',
               'asset_form',
               'asset_delay'
               ]

#%%
assets = pd.DataFrame(assets_temp,columns=asset_clomuns)
```

when I view the variable assets,something wrong 

python 2,7,11,0 and spyder 2.3.8

**Comment by ccordoba12** [comment]
This error occurs when you try to copy contents of a DataFrame but don't have anything selected to copy, as shown below. And it's already fixed in our master branch.

![seleccion_002](https://cloud.githubusercontent.com/assets/365293/11770728/e60ba214-a1d3-11e5-87e1-ebe4f62e2690.png)

If you want to copy an entire DataFrame, you need to do right click over `assets` in the Variable Explorer, and select the Copy option, like this:

![seleccion_001](https://cloud.githubusercontent.com/assets/365293/11770719/8adb92d2-a1d3-11e5-8cd3-0ca599553d4f.png)

**Comment by ccordoba12** [comment]
I forgot to mention that the fix will be available when Spyder 3.0 is released in a couple of months :-)

**Comment by lemontreeshy** [comment]
thanks so much
